test/fixtures_mysql_test.go

```diff
@@ -31,11 +62,11 @@ var TestModelMetadata = db_repo.Metadata{
 	},
 }
 
-func mysqlTestFixtures() []*fixtures.FixtureSet {
+func ormMysqlTestFixtures() []*fixtures.FixtureSet {
```

Comment:
actually i think, i would split the test.
This is due to testing different implementations.
Think of it like: we don't test all implementations in one file, so we should have each test in a different file.

Reply:
wont do it because of the "boot mysql multiple times during testing"-argument